### PR TITLE
notify datadog on deploy

### PR DESCRIPTION
**Pull request by snopoke**
@millerdev 

cc @benrudolph

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -994,6 +994,9 @@
     'pillowtop': ['localhost']
 }
 
+DATADOG_API_KEY = None
+DATADOG_APP_KEY = None
```
Comment (millerdev) [inline settings.py]: will every person doing a delpoy need these keys?

Comment (snopoke) [inline settings.py]: nope, the management command gets called on one of the servers in the cluster via fabric

Comment (snopoke) [inline settings.py]: https://github.com/dimagi/commcarehq-ansible/pull/229

File: corehq/apps/hqadmin/management/commands/record_deploy_success.py [hunk 2]
```diff
@@ -42,19 +52,31 @@ def handle(self, *args, **options):
...
             }))
 
+        if settings.DATADOG_API_KEY:
+            tags = ['environment:{}'.format(options['environment'])]
+            link = diff_link(STYLE_MARKDOWN, git_snapshot['diff_url'])
+            datadog_api.Event.create(
+                title="Deploy Success",
+                text=deploy_notification_text.format(diff_link=link),
+                tags=tags
+            )
```
Comment (millerdev) [inline corehq/apps/hqadmin/management/commands/record_deploy_success.py]: Looks like we only ever record deploy success. What is datadog used for?

File: settings.py (hunk 1 above)
Comment (millerdev) [inline settings.py]: ah, cool

File: corehq/apps/hqadmin/management/commands/record_deploy_success.py (hunk 2 above)
Comment (millerdev) [inline corehq/apps/hqadmin/management/commands/record_deploy_success.py]: Answered in slack: a potential replacement for newrelic

**Pull request merged**